tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -178,7 +185,7 @@ export const addExpiration = async (args: addExpirationArgs): Promise<void> => {
 
   await page.locator(linkExpiryDatepicker).evaluate(
     (datePicker: any, { newExpiryDate }): any => {
-      datePicker.__vue__.updateValue(newExpiryDate)
+      datePicker?.__vueParentComponent?.refs?.calendar.move(newExpiryDate)
```

Comment:
I think the PR (https://github.com/owncloud/web/pull/81280 that changes this is not ported to stable and I guess the PR is only for the master (for next version)

Reply:
I think its ported because ci passed

Reply:
No, it's not.
Maybe this works with existing stable code. But anyways, it's good.

Reply:
please make sure that the code is as much as possible same between master and stable branch